config/routes.rb

```diff
@@ -3,6 +3,7 @@
   get 'articles/index'
```

Comment:
Please do not use non-resourceful routing

Reply:
I need some more explanation on this